docs/variants/hardkernel_odroid_h4/recovery.md

```diff
@@ -0,0 +1,99 @@
+# Recovery
+
+## Intro
+
+The following documentation describes the process of recovering hardware from
+the brick state using an [RTE](../../transparent-validation/rte/introduction.md)
+and Dasharo open-source firmware.
+
+## Dual BIOS
```

Comment:
This section could be improved based on my experience from recovery after bad initial deployment: https://github.com/Dasharo/docs/pull/903/files#r1788722185

Reply:
https://github.com/Dasharo/docs/pull/903/commits/9fa8e3cb2e59fdb79531a08927d5452fe7742e55

Reply:
I also had issues with flashing the whole firmware, especially on top of working proprietary BIOS.